iFixit/react-commerce
# Fix type errors in `@ifixit/ui`

Reverts iFixit/react-commerce#750

Reintroduces changes from https://github.com/iFixit/react-commerce/pull/738

## QA

Do a quick browse of the site, and make sure images load as usual and there are no new console errors related to images.

@hcutrone @ardelato
